.github/workflows/build.yml

```diff
@@ -0,0 +1,67 @@
+name: Build
+on: [push, pull_request]
+jobs:
+  build:
+    strategy:
+      fail-fast: false
+      matrix:
+        ubuntu-release:
```

Comment:
Is it worth including macOS in the matrix like we had on Travis? Maybe save it for another PR?

Reply:
Yes, definitely! Maybe the same PR. Sorry I forgot to comment on that. Will do below.